spec/browser/test/mock-resource.spec.js

```diff
@@ -211,6 +211,17 @@ describe('Test lib / mock resources', () => {
       })
   })
 
+  it('returns a clone of the mocked response', async () => {
+    const response = ['foo', 'bar']
+    mockClient(client).resource('User').method('all').response(response)
+
+    const userOnFirstCall = await client.User.all().then((response) => response.data().shift())
+    const userOnSecondCall = await client.User.all().then((response) => response.data().shift())
+
+    expect(userOnSecondCall).toEqual(userOnFirstCall)
+    expect(response).toBe(response)
```

Comment:
I know what you wanted to check in this line, but this is impossible to fail. An object will always be itself, even if it has mutated since its creation.

Reply:
Fixed by changing this to `expect(response).toEqual(['foo', 'bar'])`